iothub/service/src/AmqpServiceClient.cs

```diff
@@ -17,10 +17,10 @@ namespace Microsoft.Azure.Devices
 {
     internal sealed class AmqpServiceClient : ServiceClient
     {
-        private const string StatisticsUriFormat = "/statistics/service?" + ClientApiVersionHelper.ApiVersionQueryStringDefault;
-        private const string PurgeMessageQueueFormat = "/devices/{0}/commands?" + ClientApiVersionHelper.ApiVersionQueryStringDefault;
-        private const string DeviceMethodUriFormat = "/twins/{0}/methods?" + ClientApiVersionHelper.ApiVersionQueryStringDefault;
-        private const string ModuleMethodUriFormat = "/twins/{0}/modules/{1}/methods?" + ClientApiVersionHelper.ApiVersionQueryStringDefault;
+        private const string StatisticsUriFormat = "/statistics/service?" + ClientApiVersionHelper.ApiVersionQueryString;
+        private const string PurgeMessageQueueFormat = "/devices/{0}/commands?" + ClientApiVersionHelper.ApiVersionQueryString;
+        private const string DeviceMethodUriFormat = "/twins/{0}/methods?" + ClientApiVersionHelper.ApiVersionQueryString;
+        private const string ModuleMethodUriFormat = "/twins/{0}/modules/{1}/methods?" + ClientApiVersionHelper.ApiVersionQueryString;
         private static readonly TimeSpan s_defaultOperationTimeout = TimeSpan.FromSeconds(100);
```

Comment:
I rem this had some effect on e2e as well? Isn't there anything to be cleaned up ?

Reply:
You remember correctly, there was a flag in the e2e tests around this. I've pushed changes to remove that flag now

Reply:
👍